scheduling-manager/src/cucumber/resources/features/scheduling/replace_after_terminate.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Deploying a Daemon environment to a cluster with Replace After Terminate
```

Comment:
should we move this to the integ-test package?

Reply:
As mentioned in the comment below, these aren't actually integration tests, they're unit tests written in cucumber so that we can use tables for these very data-driven test cases.